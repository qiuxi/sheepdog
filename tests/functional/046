#!/bin/bash

# Test vdi deletion

. ./common

for i in `seq 0 5`; do
    _start_sheep $i
done

_wait_for_sheep 6

_cluster_format -c 6

echo "delete snapshot image before current one"
_vdi_create test 4G
$DOG vdi snapshot test -s snap
_vdi_list

$DOG vdi delete test -s snap
$DOG vdi delete test -s snap # error
_vdi_list

$DOG vdi delete test
$DOG vdi delete test # error
_vdi_list

echo "delete current image before snapshot one"
_vdi_create test 4G
$DOG vdi snapshot test -s snap
_vdi_list

$DOG vdi delete test
$DOG vdi delete test # error
_vdi_list

$DOG vdi delete test -s snap
$DOG vdi delete test -s snap # error
_vdi_list
