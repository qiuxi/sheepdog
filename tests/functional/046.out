QA output created by 046
using backend plain store
delete snapshot image before current one
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
s test         1  4.0 GB  0.0 MB  0.0 MB DATE   7c2b25     6          snap
  test         0  4.0 GB  0.0 MB  0.0 MB DATE   7c2b26     6              
Cannot get VDI info for test 0 snap: Failed to find requested tag
Failed to open VDI test
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
  test         0  4.0 GB  0.0 MB  0.0 MB DATE   7c2b26     6              
Cannot get VDI info for test 0 : No VDI found
Failed to open VDI test
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
delete current image before snapshot one
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
s test         1  4.0 GB  0.0 MB  0.0 MB DATE   7c2b25     6          snap
  test         0  4.0 GB  0.0 MB  0.0 MB DATE   7c2b27     6              
Cannot get VDI info for test 0 : No VDI found
Failed to open VDI test
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
s test         1  4.0 GB  0.0 MB  0.0 MB DATE   7c2b25     6          snap
Cannot get VDI info for test 0 snap: No VDI found
Failed to open VDI test
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
